### PR TITLE
Standardize the import path in the docs

### PR DESCRIPTION
**Pull request by blakewilson**
## Description

This PR standardizes the import path in the docs to use the project root, and explains that this should be configured in `tsconfig.json`. Guides and ref docs also now display a notice saying:

> This doc assumes you're storing your pages/styles/etc. in a directory called `src`, and you have the `baseUrl` option set to `src` in your `tsconfig.json` for import modules from the root.

To be more clear on how imports are handled.

## Documentation Changes
1. Added Base Url notice to all docs that use imports from the root
2. Standardized the import paths to use no relative import paths and instead import from the root. e.g. `import 'styles/globals.css'` instead of `import '../styles/globals.css'`
3. Added a [section](https://hcixzyt38dn5ak04xxcqc36lf.js.wpenginepowered.com/docs/tutorial/basic-headless-site#setup-typescript) to the tutorial for PHP developers to configure this option in `tsconfig.json`

## Staging URL
https://hcixzyt38dn5ak04xxcqc36lf.js.wpenginepowered.com/

### CODE REVIEW

File: internal/website/docs/next/reference/expected-url-params.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ For example, say you have a page:
 Since `[postSlug]` is one of the params we look for, you can use the `usePost()` hook without any arguments and it will understand that it needs to get a post using the value stored in the `postSlug` param!
 
 ```tsx title=/src/pages/posts/[postSlug].tsx {6}
-import { client, getNextStaticProps } from '@wpengine/headless-next';
+import { client, getNextStaticProps } from '@faust/next';
```
Comment (apmatthews) [inline internal/website/docs/next/reference/expected-url-params.mdx:32]:
```suggestion
import { client, getNextStaticProps } from '@faustjs/next';
```

File: internal/website/docs/next/reference/expected-url-params.mdx [hunk 2]
```diff
@@ -48,7 +48,7 @@ export default function Page() {
 The same can be done for `[postId]` only this time it will make a request using the `postId` param to get a post by ID.
 
 ```tsx title=/src/pages/posts/[postId].tsx {6}
-import { client, getNextStaticProps } from '@wpengine/headless-next';
+import { client, getNextStaticProps } from '@faust/next';
```
Comment (apmatthews) [inline internal/website/docs/next/reference/expected-url-params.mdx:51]:
```suggestion
import { client, getNextStaticProps } from '@faustjs/next';
```

File: internal/website/src/mdx/base-url-notice.mdx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+:::tip
+
+This doc assumes you're storing your pages/styles/etc. in a directory called `src`, and you have the `baseUrl` option set to `src` in your `tsconfig.json` for import modules from the root.
```
Comment (apmatthews) [inline internal/website/src/mdx/base-url-notice.mdx:3]:
```suggestion
This doc assumes you're storing your pages/styles/etc. in a directory called `src`, and you have the `baseUrl` option set to `src` in your `tsconfig.json` for importing modules from the root.
```

**Review by apmatthews — COMMENTED** [review]
Just a couple of typo suggestions to address, but otherwise looks great!

File: internal/website/docs/next/reference/expected-url-params.mdx (hunk 1 above)
Comment (blakewilson) [reply internal/website/docs/next/reference/expected-url-params.mdx:32]: Haha, gotta love fixing a typo while introducing a new typo. Thanks for catching this.

**Pull request merged**